.github/workflows/policy-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Policy Container
+
+on:
+  push:
+    branches:  # Release only when main advances
+      - main
+
+jobs:
+  build_publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4.1.1
+
+      - name: Generate Image Name
+        run: echo IMAGE_REPOSITORY=ghcr.io/$(echo "${{ github.repository }}-policy" | tr '[:upper:]' '[:lower:]' | tr '[_]' '[\-]') >> $GITHUB_ENV
+
+      - name: Generate Image Tag  # :<tag> or :latest if not tagged
+        run: echo IMAGE_TAG=$(echo "${{ github.ref_type }} = 'tag' && ${{ github.ref_name }} || 'latest'" | tr '[:upper:]' '[:lower:]' | tr '[_]' '[\-]') >> $GITHUB_ENV
+
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Setup OPA
+        uses: open-policy-agent/setup-opa@v2
+        with:
+          version: latest
+      
+      - name: Build & Publish OPA Policy
+        run: opa build ./org-policy
+
+      - name: Publish OPA Policy
+        run: oras push ${{ env.IMAGE_REPOSITORY }}:${{ env.IMAGE_TAG }} --config ./org-policy/config.json:application/vnd.oci.image.config.v1+json bundle.tar.gz:application/vnd.oci.image.layer.v1.tar+gzip
```

Comment:
Should add an `if` here that is similar to the one on in `bundler-container`, e.g.:
```
${{ !(github.event_name == 'push' && startsWith(github.ref, 'refs/tags')) }}
```